azure-container-apps/README.md

```diff
@@ -0,0 +1,371 @@
+6. Hover over your **op-scim-bridge** container and select it, then click **Edit**.
+7. Scroll down to the environment variables and add a new one. Then fill out the following fields:
+      1. **Name**: Enter `OP_WORKSPACE_CREDENTIALS`.
+      2. **Source**: Choose **Reference a secret**.
+      3. **Value**: Select the `workspace-credentials` secret.
+
+8. Create another new variable and fill out the following fields:
+      1. Name: **OP_WORKSPACE_SETTINGS**
+      2. **Source**: Choose **Reference a secret**.
+      3. **Value**: Select the `workspace-settings` secret.
+
+9. Click **Save**, then click **Create**.
+
+</details>
+
+<hr>
+
+## Update your SCIM bridge
+
+> [!TIP]
+> Check for 1Password SCIM Bridge updates on the [SCIM bridge release page](https://app-updates.agilebits.com/product_history/SCIM).
+
+You can update your SCIM bridge in the [Azure Cloud Shell](#in-the-azure-cloud-shell-or-az-cli) or the [Azure Portal](#in-the-azure-portal).
+
+### In the Azure Cloud Shell or AZ CLI
+
+[Learn how to update 1Password SCIM Bridge in the Cloud Shell or AZ CLI.](https://support.1password.com/scim-update/#azure-container-apps)
+
+### In the Azure Portal
+
+1. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), Select **Containers** from the left hand side.
+2. Click **Edit and deploy**.
+3. Put a check in the box next to your **op-scim-bridge** container and select **Edit**.
+4. Change the version number **2.9.0** in the **Image and Tag** field, **1password/scim:v2.9.0** to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
+5. Select **Save**.
+6. Select **Create** to deploy a new revision using the updating image.
+7. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).
+
+## Appendix: Resource recommendations
+
+The pod for 1Password SCIM Bridge should be vertically scaled you provision a large number of users or groups. These are our default resource specifications and recommended configurations for provisioning at scale:
+
+| Volume    | Number of users | CPU   | memory |
+| --------- | --------------- | ----- | ------ |
+| Default   | <1,000          | 0.25  | 0.5Gi  |
+| High      | 1,000–5,000     | 0.5   | 1.0Gi  |
+| Very high | >5,000          | 1.0   | 1.0Gi  |
+
+If you're provisioning more than 1,000 users, update the resources assigned to the SCIM bridge container to follow these recommendations. The resources specified for the Redis container do not need to be adjusted.
+
+### Default
+
+To revert to the default specification that is suitable for provisioning up to 1,000 users, run the following command:
```

Comment:
@Penderworth Looking at this line again, I think you were right to go with "up to 1,000" users here, since the next option is _between_ 1,000 - 5,000. Please disregard my "fewer than" suggestion.